### ISSUE TITLE
Wizard fails when pointing at sencha-touch-2.0.0-pr4

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#63]
```
There will be a copy file error and the project will not be completely created.

Most notably, plugins.xml is not created and AndroidManifest.xml is not set up.

Workaround: In the Sencha install, copy sencha-touch-all-debug-w-comments.js to 
sencha-touch-debug-w-comments.js

This bug will be fixed in 1.2.7, due out in February 2012.




```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 28 Jan 2012 at 12:46

**Comment by GoogleCodeExporter** [comment]
```
See also Issue #64
```

Original comment by `paul.beu...@gmail.com` on 28 Jan 2012 at 12:50

**Comment by GoogleCodeExporter** [comment]
```
Released with 1.2.7 February 7, 2012
```

Original comment by `paul.beu...@gmail.com` on 7 Feb 2012 at 4:10
- Changed state: **Fixed**
